app/presentation/RoomItem/LastMessage.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Text } from 'react-native';
+import { emojify } from 'react-emojione';
+import PropTypes from 'prop-types';
+
+import I18n from '../../i18n';
+import styles from './styles';
+
+const formatMsg = ({
+	lastMessage, type, showLastMessage, username
+}) => {
+	if (!showLastMessage) {
+		return '';
+	}
+	if (!lastMessage) {
+		return I18n.t('No_Message');
+	}
+
+	let prefix = '';
+	const me = lastMessage.u.username === username;
+
+	if (!lastMessage.msg && Object.keys(lastMessage.attachments).length > 0) {
+		if (me) {
+			return I18n.t('User_sent_an_attachment', { user: I18n.t('You') });
+		} else {
+			return I18n.t('User_sent_an_attachment', { user: lastMessage.u.username });
+		}
+	}
+
+	if (me) {
+		prefix = I18n.t('You_colon');
+	}	else if (type !== 'd') {
+		prefix = `${ lastMessage.u.username }: `;
+	}
+
+	let msg = `${ prefix }${ lastMessage.msg.replace(/[\n\t\r]/igm, '') }`;
+	if (msg) {
+		msg = emojify(msg, { output: 'unicode' });
+	}
+	return msg;
```

Comment:
better use `return msg && emojify(msg, { output: 'unicode' });` and remove condition on 37 line; and then make msg const